sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/AppConfigurationReplicaClientsBuilder.java

```diff
@@ -230,35 +219,36 @@ private AppConfigurationReplicaClient modifyAndBuildClient(ConfigurationClientBu
         for (String profile : environment.getActiveProfiles()) {
             if ("dev".equalsIgnoreCase(profile)) {
+                // TODO (mametcal) this no longer works
                 this.isDev = true;
                 break;
             }
         }
-        this.env = environment;
     }
 
     protected ConfigurationClientBuilder createBuilderInstance() {
+        // TODO (mametcal) Need global properties to work.
         RetryStrategy retryStatagy = null;
 
-        String mode = env.getProperty(AzureGlobalProperties.PREFIX + "." + RETRY_MODE_PROPERTY_NAME);
-        String modeService = env.getProperty(AzureAppConfigurationProperties.PREFIX + "." + RETRY_MODE_PROPERTY_NAME);
+        //String mode = env.getProperty(AzureGlobalProperties.PREFIX + "." + RETRY_MODE_PROPERTY_NAME);
+        //String modeService = env.getProperty(AzureAppConfigurationProperties.PREFIX + "." + RETRY_MODE_PROPERTY_NAME);
 
-        if ("exponential".equals(mode) || "exponential".equals(modeService) || (mode == null && modeService == null)) {
-            Function<String, Integer> checkPropertyInt = parameter -> (Integer.parseInt(parameter));
-            Function<String, Duration> checkPropertyDuration = parameter -> (DurationStyle.detectAndParse(parameter));
+        //if ("exponential".equals(mode) || "exponential".equals(modeService) || (mode == null && modeService == null)) {
+        Function<String, Integer> checkPropertyInt = parameter -> (Integer.parseInt(parameter));
+        Function<String, Duration> checkPropertyDuration = parameter -> (DurationStyle.detectAndParse(parameter));
 
-            int retries = checkProperty(MAX_RETRIES_PROPERTY_NAME, defaultMaxRetries,
-                " isn't a valid integer, using default value.", checkPropertyInt);
+        int retries = checkProperty(MAX_RETRIES_PROPERTY_NAME, defaultMaxRetries,
+            " isn't a valid integer, using default value.", checkPropertyInt);
 
-            Duration baseDelay = checkProperty(BASE_DELAY_PROPERTY_NAME, DEFAULT_MIN_RETRY_POLICY,
-                " isn't a valid Duration, using default value.", checkPropertyDuration);
-            Duration maxDelay = checkProperty(MAX_DELAY_PROPERTY_NAME, DEFAULT_MAX_RETRY_POLICY,
-                " isn't a valid Duration, using default value.", checkPropertyDuration);
+        Duration baseDelay = checkProperty(BASE_DELAY_PROPERTY_NAME, DEFAULT_MIN_RETRY_POLICY,
+            " isn't a valid Duration, using default value.", checkPropertyDuration);
+        Duration maxDelay = checkProperty(MAX_DELAY_PROPERTY_NAME, DEFAULT_MAX_RETRY_POLICY,
+            " isn't a valid Duration, using default value.", checkPropertyDuration);
 
-            retryStatagy = new ExponentialBackoff(retries, baseDelay, maxDelay);
-        }
+        retryStatagy = new ExponentialBackoff(retries, baseDelay, maxDelay);
+        //}
 
-        ConfigurationClientBuilder builder = clientFactory.build();
```

Comment:
Fix this

Reply:
Updated, now the properties under the `spring.cloud.azure.retry, credential`, etc will be taken for `AppConfigurationReplicaClientsBuilder`.

Reply:
@mrm9084  please take a look, thanks!